config/locales/en.models.yml

```diff
@@ -267,3 +270,7 @@ en:
         remittance_information: Bill To
       nufs_account:
         account_number: Payment Source Number
+      training_request:
+        user: Name
+        product: Instrument
```

Comment:
It doesn't look like bundles are ever restricted, however it might be possible to have a restricted item within a bundle; I'm not sure how that's handled.

```
Product.where(requires_approval: true).active_plus_hidden.count(group: 'type')
   (3.1ms)  SELECT COUNT(*) AS count_all, type AS type FROM "PRODUCTS" WHERE "PRODUCTS"."REQUIRES_APPROVAL" = 1 AND "PRODUCTS"."IS_ARCHIVED" = 0 GROUP BY type
 => {"Instrument"=>252, "Item"=>7, "Service"=>52}
```

I don't think we need to worry about it.